README.md

```diff
@@ -1,8 +1,113 @@
 # bazel_rules_hdl
 Hardware Description Language (Verilog, VHDL, Chisel, nMigen, etc) with open tools (Yosys, Verilator, OpenROAD, etc) rules for Bazel (https://bazel.build)
- 
+
+**THIS REPO REQUIRES BAZEL 3.5.0 or greater**
+
+## Installation
+
+In your `WORKSPACE` file. Which is a file at the top directory of every bazel
+repo:
+
+```starlark
+git_hash = "{LATEST_HASH}"
+git_sha256 = "{LATEST_SHA}"
```

Comment:
Tiny nit, feel free to ignore: I would prefix these variables with `rules_hdl_`, because I'm sure that in most cases where this is used the term "git hash" could potentially refer to several different repos.